change main readme file (#45)
layerml/examples: README.md

✅ 2021-11-17 08:51:15 | featureset  passenger_features             ━━━━━━━━━━━━━━━━━━━━━━ DONE      [34152ms]                                     
                                     https://beta.layer.co/features/420bf9a1-d114-4fed-ba4c-d37abf54e532                                           
✅ 2021-11-17 08:51:50 | model       survival_model                 ━━━━━━━━━━━━━━━━━━━━━━ DONE      [24019ms]                                     
                                     https://beta.layer.co/models/00050c3d-d6e1-4db0-9767-c5091309fcd3/trains/5def9683-5ccf-477e-b0b5-56d56d3b3623 
LAYER RUN SUCCEEDED in 61166ms

```
Layer runs your project and places the generated entities in a [data](https://docs.beta.layer.co/docs/data/data-catalog) and [model catalog](https://docs.beta.layer.co/docs/modelcatalog). 

## The data catalog
Layer Data Catalog is a central repository for your feature sets so that they can be systematically built, monitored, and evaluated to reduce the cost of errors. 
The first link generated on the terminal will take you to the data catalog:

![Data Catalog](images/datacatalog.png)

> ## The model catalog

Reviewer: ```suggestion
## Layer Model Catalog
```

Layer Model Catalog provides a centralized, managed, indexed storage space for ML models. It ensures that model artifacts are versioned and immutable

> The second link that was generated on the terminal takes you to the model catalog:

Reviewer: ```suggestion
The second link that was generated on the terminal takes you to the Model Catalog:
```

![Model Catalog](images/modelcatalog.png)

## Deploying the model 

> You can deploy your model by clicking the Deploy button next to the model on the model catalog. 

Reviewer: ```suggestion
You can deploy your model by clicking the Deploy button next to the model in the Model Catalog. 
```

![Deploy Model](images/deploy.png)

Once the deployment process is complete, you will get an
API end point that you can use make predictions on the model. 

> ## Run predictions on the deployed model 

Reviewer: ```suggestion

## Run predictions on the deployed model 
```

Click the 🔗 icon to copy the deployment URL. You can then use this URL to run predictions on the model: 

![Deployment URL](images/deployment_url.png)

```yaml
curl --header "Content-Type: application/json; format=pandas-records" \
  --request POST \
  --data '[{"Sex":0, "EmbarkStatus":0, "Title":0, "IsAlone":0, "AgeBand":1, "FareBand":2}]' \
  $MODEL_DEPLOYMENT_URL
```

> The API returns either a [0] or a [1], predicting whether that passenger survived.

Reviewer: ```suggestion

The API returns either a [0] or a [1], predicting whether that passenger survived.
```

> ## Use the generated entities in a Jupyter Notebook

Reviewer: ```suggestion

## Use the generated entities in a Jupyter Notebook
```

> Entities generated with Layer can also be accessed in a Jupyter Notebook. Layer allows you to access the datasets, featuresets and models. 

Reviewer: ```suggestion
Entities generated with Layer can also be accessed in a Jupyter Notebook. Layer allows you to access the datasets, feature sets, and models. 
```
